quickstart/k8s-the-prod-way/app.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Provisioning a Kubernetes Application
+---
+
+This lab assumes you have successfully completed [lab 1](./identity.html), and [lab
+2](./infrastructure.html) the results of which are to have (1) successfully bootstrapped the
+identity platform for CI/CD and teams, and (2) successfully created the Kubernetes cluster on which
+we will deploy.
+
+In this tutorial, we will use this identity to:
+
+* Use the identity provisioned in lab 1 and the Kubernetes cluster provisioned in lab 2 to provision
+  a Kubernetes app, Wordpress.
+
+Inside the [Kubernetes the Prod Way repository][ktpw], the `services` directory contains the
```

Comment:
I don't see a `services/` subdirectory at https://github.com/pulumi/kubernetes-the-prod-way